frontend/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@cycle/core": "^5.0.1",
     "@cycle/dom": "^7.1.1",
+    "@cycle/isolate": "^1.2.0",
```

Comment:
it's just adding randomize class to every components instance - to avoid scripts and styles conflicts.